caffe2/contrib/tensorrt/tensorrt_tranformer.cc

```diff
@@ -40,6 +40,21 @@ std::unordered_map<std::string, TensorShape> InferShapes(
   return shape_hints;
 }
 
+void DumpModel(const ::ONNX_NAMESPACE::ModelProto& model) {
+  std::ofstream ff("trt.onnx");
```

Comment:
Fixed model name is problematic if multiple ONNX subgraphs are extracted from a Caffe2 model and offloaded to TRT. Please at least add an index.